packages/web3-common/src/web3_base_provider.ts

```diff
@@ -6,18 +6,15 @@ import {
 	JsonRpcResult,
 	Web3APIMethod,
 	Web3APISpec,
+	JsonRpcNotification,
+	JsonRpcSubscriptionResult,
 } from './types';
 
-export interface ProviderMessage<T = JsonRpcResult> {
-	type: string;
-	data: T;
-}
-
 // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1193.md#connectivity
 export type Web3BaseProviderStatus = 'connecting' | 'connected' | 'disconnected';
```

Comment:
These types should be in types.ts file:
```
Web3BaseProviderStatus
Web3BaseProviderCallback
```

Reply:
Yes I agree to this change, will include in this PR. Would be nice to narrow down such points in the relevant PRs.